docs/_blog/liana-payjoin.md

```diff
@@ -0,0 +1,36 @@
+
+For instance, imagine a policy with two keys: a primary key for day-to-day spending and a recovery key to regain access if the primary key is lost. If the user loses the primary key, the recovery key can be used to access the funds—but only after the timelock expires.
+
+However, this approach introduces a new challenge: most users aren't actively spending their UTXOs. [A 2024 study](https://www.sciencedirect.com/science/article/pii/S0378437124008045) found that Bitcoin UTXO age follows a power-law distribution. In practice, this means users either spend their coins shortly after receiving them or leave them untouched for extended periods. And [the majority of UTXOs are not spent for months or years](https://charts.bitbo.io/hodl-waves/). To avoid triggering the recovery path unintentionally, many users perform a self-spend or reconciliation.
+
+<img alt="Liana Timelocked UTXOs" src="../assets/liana-coins.png" />
+
+Reconciliation, however, introduces a privacy risk. It creates an on-chain footprint that allows surveillance firms to estimate a user's wallet balance and transaction history. This is primarily due to the [common input ownership heuristic](https://en.bitcoin.it/wiki/Common-input-ownership_heuristic), which assumes all inputs in a transaction belong to the same entity. Combined with unique transaction patterns that act like fingerprints, these heuristics allow on-chain analysts to cluster a user's UTXOs and glean sensitive information.
+
+## Payjoin as a Privacy-Preserving Refresh
+
+This is where Payjoin becomes valuable. It allows Liana users to refresh or reconcile soon-to-expire UTXOs during normal payments. The same method can apply to self-spends. If a user isn’t expecting incoming payments, they can still use Payjoin the next time they send a transaction. Whether as sender or receiver, users benefit from reduced fees and a more obfuscated transaction graph—preserving privacy while maintaining the integrity of their recovery policy.
+
+<img alt="Liana Payjoin" src="../assets/liana-batch-payment.png" />
+
+## Caveats
+
+It's important to note that there is no privacy gain when using Payjoin with P2WSH descriptors unless both the sender and receiver share the same or structurally similar descriptor. This is because the spending script is revealed on-chain, allowing observers to fingerprint the wallet based on its script template. If the participants use different descriptors, their inputs become trivially distinguishable, defeating the purpose of Payjoin. Taproot-based descriptors using Miniscript do not suffer from this issue. With Taproot, key-path spending hides the recovery script entirely unless deliberately revealed, enabling privacy-preserving Payjoin transactions even between users with different wallet policies.
+
```

Comment:
"privacy gain" is another term I like to avoid. I prefer something like "...the ability to break common input ownership is limited when..."